src/main/java/graphql/schema/validation/ValidationRule.java

```diff
@@ -1,9 +1,11 @@
 package graphql.schema.validation;
 
 import graphql.schema.GraphQLFieldDefinition;
+import graphql.schema.GraphQLType;
 
 public interface ValidationRule {
```

Comment:
We should rename this to `SchemaValidationRule` or something similar ... `ValidationRule` too generic